assets/config/0.5.2-coins.json

```diff
@@ -3663,7 +3663,7 @@
     "coinpaprika_id": "tkl-tokel",
     "coingecko_id": "test-coin",
     "electrum": [
-      
+
```

Comment:
please remove the extra line adding

Reply:
Oh, what a mess, I'm sorry this got added. I'm removing.